certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,383 @@
+                    "email": "your_email_here",
+                    "subject": "TEST firma remota"
+                  }
+                },
+                "document": {
+                  "templateCode": "templateSample"
+                },
+                "callbackMails": "your_email_here"
+              },
+              "required": [
+                "workflow",
+                "document",
+                "groupCode",
+                "notification"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/documents/api/v3/messages/dispatch": {
+      "post": {
+        "summary": "Create sign request",
+        "description": "Create a sign request and get the permanent link",
+        "operationId": "CreateSignRequest",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json\r",
+            "description": "Content-Type",
+            "x-ms-summary": "Content-Type"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*/*\r",
+            "description": "Accept",
+            "x-ms-summary": "Accept"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "groupCode": {
+                  "type": "string",
+                  "description": "groupCode",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "WhoIAm",
+                    "value-path": "groups-key-item-output",
+                    "value-collection": "groups",
+                    "value-title": "groups-key-item-output"
+                  }
+                },
+                "workflow": {
+                  "type": "object",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "type",
+                      "title": "",
+                      "default": "PRESENTIAL",
+                      "x-ms-visibility": "internal"
+                    }
+                  },
+                  "description": "workflow",
+                  "required": [
+                    "type"
+                  ]
+                },
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "templateCode": {
+                      "type": "string",
+                      "description": "templateCode",
+                      "title": ""
```

Comment:
@anfuca Empty property values can be removed until unless you have a specific reason to keep them.